src/components/UserProfile/UserProfile.tsx

```
@@ -0,0 +1,471 @@
+              <div className="position-relative">
+                <small
+                  className="text-muted fw-medium"
+                  style={{
+                    fontSize: '12px',
+                    color: '#6c757d',
+                    cursor: 'pointer',
+                    padding: '2px 4px',
+                    borderRadius: '4px',
+                    transition: 'background-color 0.2s',
+                  }}
+                  role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)